circuit_knitting_toolbox/circuit_cutting/cutting_decomposition.py

```diff
@@ -86,3 +93,123 @@ def partition_circuit_qubits(
         new_qc.data[i] = CircuitInstruction(qpd_gate, qubits=qubit_indices)
 
     return new_qc
+
+
+def decompose_gates(circuit: QuantumCircuit, gate_ids: Sequence[int]) -> QuantumCircuit:
+    r"""
+    Transform specified gates into :class:`TwoQubitQPDGate`\ s.
+
+    Args:
+        circuit: The circuit containing gates to be decomposed
+        gate_ids: The indices of the gates to decompose
+
+    Returns:
+        A copy of the input circuit with the specified gates replaced with :class:`TwoQubitGate`\ s
+    """
+    # Replace specified gates with TwoQubitQPDGates
+    new_qc = circuit.copy()
```

Comment:
Thoughts on adding an `inplace` kwarg?  It can default to `False`, and then we would get the current behavior.  Not urgent; we can always do this later if we wish.

Reply:
Done. Added one for partition_circuit_qubits as well